scripts/hid_configurator/configurator_core.py

```diff
@@ -639,5 +646,46 @@ def get_dfu_image_version(dfu_image):
     return ver
 
 
+class Step:
+    def __init__(self, r, g, b, substep_count, substep_time):
+        self.r = r
+        self.g = g
+        self.b = b
+        self.substep_count = substep_count
+        self.substep_time = substep_time
+
+
+def led_send_single_step(dev, recipient, step, led_id):
+    event_id = (EVENT_GROUP_LED_STREAM << GROUP_FIELD_POS) \
+               | (LED_STREAM_DATA << TYPE_FIELD_POS)
+
+    event_data = struct.pack('<BBBHHB', step.r, step.g, step.b,
```

Comment:
``<BBBHHB`` is a magic string.

Reply:
It is format used by struct module in Python:
https://docs.python.org/3/library/struct.html

Reply:
magic sequence - provide infromation about origin/meaning of that in a comment